librz/include/rz_sign.h

```diff
@@ -2,165 +2,14 @@
 #define RZ_SIGN_H
```

Comment:
this file is almost empty now. is it used only to include rz_flirt.h?

Reply:
yes, and the header for the version.

Reply:
i reverted it back as you have asked.